data/Makefile.am

```diff
@@ -39,8 +39,34 @@ install-sample-data:
 	rm fedora-20-icons.tar.gz; \
 	cd -
 
+com.endlessm.SystemExternalApps.rules: com.endlessm.SystemExternalApps.rules.in
+	$(AM_V_GEN) $(SED)					\
+	-e "s|\@privileged_group\@|$(PRIVILEGED_GROUP)|"	\
+	-e "s|\@helper_user\@|$(EXT_APPS_HELPER_USER)|"		\
```

Comment:
Cool. Thanks for the hint. (I have copied most of the autotools stuff from Flatpak)